com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py

```diff
@@ -494,7 +504,7 @@ def _distributed_test(self, config, username, password):
         if sjr['return_code'] != 0:
             _logger.error("Failed to submit job to distributed instance.")
             return False
-        return self._distributed_wait_for_result()
+        return self._distributed_wait_for_result(stc.ContextTypes.DISTRIBUTED)
```

Comment:
This seems a random change? Not seeing a corresponding change when the context is the service?